src/main/java/io/github/syst3ms/skriptparser/parsing/SyntaxParser.java

```diff
@@ -120,6 +127,18 @@
                 return variable;
             }
         }
+        // This is to prevent us from exchanging boolean operators with lists.
+        if (s.toLowerCase().startsWith("list ")) {
+            s = s.substring("list ".length());
+        } else {
+            // We parse boolean operators first to prevent clutter while parsing.
```

Comment:
If parsing has failed here, there is no need to try to parse it later down the line. Perhaps there should be a flag to ignore ExprBooleanBoolean operators in that case?

Reply:
I don't get what you mean by that entirely.